text/0000-build-std/3-motivation.md

```diff
@@ -0,0 +1,97 @@
+# Motivation
+[motivation]: #motivation
+
+While the pre-built standard library has been sufficient for the majority of
+Rust users, there are a variety of use-cases which require the ability to
+re-build the standard library.
```

Comment:
As written, this makes it sound like these are Motivations that this RFC is attempting to address, rather than focusing on the MVP motivations and leaving the rest to future possibilities.

Reply:
I'll adjust the wording here. In the full proposal, this paragraph is changed and the motivations are more clearly split up into "we're aiming to enable this", "we're not enabling this now but we want it to be doable in future" and "we don't care if this is impossible after these changes". I just wrote this quickly when splitting the context parts into separate commits.

Reply:
Should be addressed in 8765bc5861a8ba6ccb31601b625ba88fb4420125

Reply:
I think the disclaimer should be more prominent, like a `note` or a parenthetical / italicized text before anything else.   As is, that is important context for understanding the whole section, especially as it deviates from most RFCs, and it can easily be unnoticed where it is at.

Is there a reason we are including everything?  The original document did that as well and I thought we were overall preferring for focusing the Motivation on the MVP or to at least break out the MVP and all into two separate lists.